06-building-a-program-start-to-finish/writing-tests/output/spec.js

```diff
@@ -151,9 +157,11 @@ describe('InvertedIndex validateFile', () => {
   });
 });
 
-describe('InvertedIndex cleanValues', () => {
-  it('Should return an array of clean values', () => {
+describe('InvertedIndex cleanValues Method', () => {
+
+  it('Should return an array of clean keywords', () => {
     expect(InvertedIndex.cleanValues('How, are, you doing today'))
       .toEqual(['how', 'are', 'you', 'doing', 'today']);
   });
+  
```

Comment:
Please go through your tests again. You have eslint violation errors (Max-len, Trailing spaces, Unnecessary line breaks after in your `describe` function just before your `it` function) and also ensure that your tests are all right.